scripts/alert_emails/send_emails.ts

```diff
@@ -7,6 +7,14 @@ import { generateAlertEmailData, readAlerts, resolveSnapshot } from './utils';
 
 const BATCH_SIZE = 20;
 
+interface SnapshotStats {
+  sent: Date;
+  locationsWithAlerts: number;
+  totalEmailsSent: number;
+  uniqueEmailAddresses: number;
+  invalidEmailsRemoved: number;
+}
```

Comment:
This is the other part of the "fix" in that going forward we're going to collect stats after every email alert run so that we have per-snapshot stats and a global total of emails sent.